More failures on Windows 2022 with renaming. Not related, but CI is flaky on Windows 😞 

```
`=== FAIL: github.com/docker/docker/integration-cli TestDockerSuite/TestContainerAPIChunkedEncoding (0.12s)
    check_test.go:170: assertion failed: error is not nil: Error response from daemon: container 7819ec9f095a635eeb94806db9cbc767eef82837785085a202548c3e809b93c0: driver "windowsfilter" failed to remove root filesystem: failed to detach VHD: failed to detach virtual disk: The device is not ready.: rename D:\CI\PR-43138\2\daemon\windowsfilter\7819ec9f095a635eeb94806db9cbc767eef82837785085a202548c3e809b93c0 D:\CI\PR-43138\2\daemon\windowsfilter\7819ec9f095a635eeb94806db9cbc767eef82837785085a202548c3e809b93c0-removing: Access is denied.: failed to remove 7819ec9f095a635eeb94806db9cbc767eef82837785085a202548c3e809b93c0
    --- FAIL: TestDockerSuite/TestContainerAPIChunkedEncoding (0.12s)
```

--- a/Dockerfile.windows
+++ b/Dockerfile.windows
@@ -166,7 +166,7 @@ FROM microsoft/windowsservercore
 SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
 
 ARG GO_VERSION=1.17.5
-ARG CONTAINERD_VERSION=1.5.8
+ARG CONTAINERD_VERSION=1.5.9
 ARG GOTESTSUM_VERSION=v1.7.0
 
 # Environment variable notes:

--- a/hack/dockerfile/install/containerd.installer
+++ b/hack/dockerfile/install/containerd.installer
@@ -15,7 +15,7 @@ set -e
 # the binary version you may also need to update the vendor version to pick up
 # bug fixes or new APIs, however, usually the Go packages are built from a
 # commit from the master branch.
-: "${CONTAINERD_VERSION:=v1.5.8}"
+: "${CONTAINERD_VERSION:=v1.5.9}"
 
 install_containerd() (
 	echo "Install containerd version $CONTAINERD_VERSION"

